ibis/expr/types.py

```diff
@@ -1247,7 +1248,7 @@ def literal(value, type=None):
         return ops.Literal(value, dtype=dtype).to_expr()
 
 
-def array(values):
+def array(values, type=None):
```

Comment:
Can you add a test case for `array(..., type="some_type_string")`?

Reply:
done!